native/http-rs-axum/Kraftfile

```diff
@@ -0,0 +1,26 @@
+spec: v0.6
+
+name: http-rs-axum
+
+unikraft:
+  version: stable
+  kconfig:
+    CONFIG_LIBPOSIX_EVENTFD: 'y'
+    CONFIG_LIBPOSIX_UNIXSOCKET: 'y'
+    CONFIG_LIBUKNETDEV_EINFO_LIBPARAM: 'y'
+
+libraries:
+  musl: stable
```

Comment:
Currently it seems like it doesn't compile with `musl` stable, but works fine with `musl` staging.

Reply:
Oh, that's annoying. When I opened this PR, everything was working. Setting everything to `'0.16.0'` works too. Not sure what we should do here. @razvand, what do you think?

Reply:
I just tested again, and it works with `stable`.